### PR TITLE
duckduckgo#687 make cursor in feedback edit text visible

### PR DESCRIPTION
**Pull request by donfuxx**
Task/Issue URL: 
Tech Design URL: https://github.com/duckduckgo/Android/issues/687
CC: 

**Description**:
image

**Steps to test this PR**:
1. go to feedback form
1. type something
1. see cursor appearing

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by donfuxx** [comment]
@cmonfortep Now it has a blue cursor style: 

image

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -259,11 +259,16 @@
         <item name="android:layout_height">match_parent</item>
     </style>
 
+    <style name="BlueCursor" parent="ThemeOverlay.MaterialComponents.TextInputEditText.OutlinedBox">
```
Comment (cmonfortep) [inline app/src/main/res/values/styles.xml:262]: Can we rename this style from `BlueCursor` to `TextInputEditTextTheme`? Using `Blue` in the style name will force you to change it (and all the usages) if you change the property inside it. Better to decouple concrete values from the style name, that way we can change them under the hood without big refactors.

Comment (donfuxx) [reply app/src/main/res/values/styles.xml:262]: Hi @cmonfortep 
I am just reading this. Sounds good to me. I'll change the name here

Comment (donfuxx) [inline app/src/main/res/values/styles.xml:262]:
```suggestion
    <style name="TextInputEditTextTheme" parent="ThemeOverlay.MaterialComponents.TextInputEditText.OutlinedBox">
```

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -259,11 +259,16 @@
...
+        <item name="android:colorControlActivated">@color/cornflowerBlue</item>
+    </style>
+
     <style name="FeedbackInputBoxStyle" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox">
         <item name="boxStrokeColor">@color/cornflowerBlue</item>
         <item name="hintTextColor">@color/cornflowerBlue</item>
         <item name="android:textColorHint">?attr/feedbackEditTextHintColor</item>
         <item name="boxBackgroundColor">?attr/feedbackEditTextBackgroundColor</item>
+        <item name="android:theme">@style/BlueCursor</item>
```
Comment (donfuxx) [inline app/src/main/res/values/styles.xml:271]:
```suggestion
        <item name="android:theme">@style/TextInputEditTextTheme</item>
```

**Comment by cmonfortep** [comment]
Thanks @donfuxx This is now merged!

**Pull request merged**